src/main/scripts/src/app/core/http-json.service.ts

```diff
@@ -18,13 +18,26 @@
  * -----------------------------------------------------------------------/
  */
 
-import {Component} from '@angular/core';
+import {Injectable} from '@angular/core';
+import {Http, Headers, RequestOptions} from '@angular/http';
 
-@Component({
-  selector: 'workspace',
-  template: require('./workspace.component.html'),
-  styles: [require('./workspace.component.scss').toString()]
-})
-export class WorkspaceComponent {
+@Injectable()
+export class HttpJson {
```

Comment:
I would suggest the following changes:

1. rename this class to `HttpClient`
2. implement all HTTP methods here
3. use this class everywhere instead of Angular's `Http` class